### ISSUE TITLE
TypeError: Cannot read property 'url' of undefined

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#94]
OK, this is a bit weird (and may just be my env)...

I was testing #86 locally [using Chrome] and was stuck on the "Install the Add-on" page (since I'm in Chrome and can't install a Firefox add-on).
Since the URL for the splash page and logged-in state are both "/", I was caught between a rock and a rockier place. Naturally I tried going to http://ideatown.dev:8000/accounts/logout/ to get back into the logged out state.

Now I'm stuck at an empty page with a console error:

> ### "TypeError: Cannot read property 'url' of undefined"

```
TypeError: Cannot read property 'url' of undefined
    at render (http://ideatown.dev:8000/static/app/app.js:810:63)
    at assign._setRender.Object.defineProperty.set.fn._render (http://ideatown.dev:8000/static/app/app.js:17147:24)
    at ViewSwitcher._render (http://ideatown.dev:8000/static/app/app.js:16718:30)
    at ViewSwitcher._show (http://ideatown.dev:8000/static/app/app.js:16699:10)
    at ViewSwitcher.set (http://ideatown.dev:8000/static/app/app.js:16673:14)
    at PageManager.onNewPage (http://ideatown.dev:8000/static/app/app.js:113:28)
    at triggerEvents (http://ideatown.dev:8000/static/app/app.js:6271:59)
    at Object.Events.trigger (http://ideatown.dev:8000/static/app/app.js:6202:21)
    at landing (http://ideatown.dev:8000/static/app/app.js:165:33)
    at extend.execute (http://ideatown.dev:8000/static/app/app.js:13524:32)(anonymous function) @ main.js:49
```

(where "main.js:49" is [this](https://github.com/mozilla/idea-town/blob/2e266451530ec119fc2a11019aedf82dea25153d/idea_town/frontend/static-src/app/main.js#L49))

![idea_town](https://cloud.githubusercontent.com/assets/557895/10111767/110980bc-638b-11e5-9bde-557cacdf3bc0.png)

**Comment by jaredhirsch** [comment]
@pdehaan Are you still seeing this?

**Comment by lmorchard** [comment]
This looks fixed now, I think it had something to do with the race conditions just fixed in #142. Cleaning up some issues, feel free to reopen if this still happens
